post-payments.php

```diff
@@ -184,6 +199,7 @@ public function add_author( $post_author, $post_id, $data = null ) {
 			$data = array(
 				'posts' => array(),
 				'payments' => array(),
+                'tags' => array(),
```

Comment:
last mention of it, but it looks like the spacing is off because of spaces instead of tabs